VideoWeb/VideoWeb/ClientApp/src/assets/i18n/cy.json

```diff
@@ -982,6 +982,7 @@
     "grc---dvsa": "GRC - DVSA",
     "grc---crb": "GRC - CRB",
     "grc---charity": "GRC - Elusennau",
+    "grc---ej": "GRC - EJ",
```

Comment:
Was there confirmation from the Welsh translation team to use this? I don't see anything on the ticket

Reply:
sorry yes, I have confirmed this with Lorenzo and Abbey, I will add a comment in the ticket as well